db/dcrsqlite/blocksretrieval_test.go

```diff
@@ -48,3 +48,41 @@ func TestEmptyDBRetrieveBlockFeeInfo(t *testing.T) {
 			cmp.Diff(*result, defaultChartsData))
 	}
 }
+
+func TestEmptyDBGetBestBlockHash(t *testing.T) {
+	testutil.BindCurrentTestSetup(t)
+	db := ObtainReusableEmptyDB()
+	str := db.GetBestBlockHash()
+	if str != "" {
+		testutil.ReportTestFailed(
+			"GetBestBlockHash() failed: expected empty string, returned %v",
+			str)
+	}
+}
+
+func TestEmptyDBGetBestBlockHeight(t *testing.T) {
+	testutil.BindCurrentTestSetup(t)
+	db := ObtainReusableEmptyDB()
+	h := db.GetBestBlockHeight()
+	if h != -1 {
+		testutil.ReportTestFailed(
+			"db.GetBestBlockHeight() returned %d, expected -1",
+			h)
+	}
+}
+
+func TestEmptyDBGetStakeInfoHeight(t *testing.T) {
+	testutil.BindCurrentTestSetup(t)
+	db := ObtainReusableEmptyDB()
+	endHeight, err := db.GetStakeInfoHeight()
+	if err != nil {
+		testutil.ReportTestFailed(
+			"GetStakeInfoHeight() failed: %v",
+			err)
+	}
+	if endHeight != -1 {
+		testutil.ReportTestFailed(
+			"GetStakeInfoHeight() failed: returned %d, expected -1",
+			endHeight)
```

Comment:
My strategy for the wrapping is the following: 
Changing the error message at the line 85 wouldn't affect lines 84 and 86.
Renaming `endHeight` would affect only line 86 (and 77).
Line 84 is never changed and so on.

The goal is to make future diffs minimalistic and simple.